trio/_tests/test_signals.py

```diff
@@ -108,7 +108,7 @@ async def test_open_signal_receiver_no_starvation() -> None:
             # Clear out the last signal so that it doesn't get redelivered
             while get_pending_signal_count(receiver) != 0:
                 await receiver.__anext__()
-        except:  # pragma: no cover
+        except BaseException:  # pragma: no cover
```

Comment:
This too is a behavior change -- it might catch a ctrl+c now. That's probably unintended. (... though it *is* just a test case)

Reply:
No, bare `except:` catches anything, not just `Exception` subclasses.

Reply:
Oh oops!